test/dotnet-new3.UnitTests/FirstRunTest.cs

```diff
@@ -1,43 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
let's keep this test - it's useful even without first run to test default installation.